docs/content/configuration/historical.md

```diff
@@ -31,7 +31,7 @@ The historical node uses several of the global configs in [Configuration](../con
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.segmentCache.locations`|Segments assigned to a Historical node are first stored on the local file system (in a disk cache) and then served by the Historical node. These locations define where that local cache resides. This value cannot be NULL or EMPTY. Here is an example `druid.segmentCache.locations=[{"path": "/mnt/druidSegments", "maxSize": 10000}]`.| none |
+|`druid.segmentCache.locations`|Segments assigned to a Historical node are first stored on the local file system (in a disk cache) and then served by the Historical node. These locations define where that local cache resides. This value cannot be NULL or EMPTY. Here is an example `druid.segmentCache.locations=[{"path": "/mnt/druidSegments", "maxSize": 10000}, "freeSpacePercent": 1.0]`. "freeSpacePercent" is optional, if provided then enforces that much of free disk partition space while storing segments.| none |
```

Comment:
Can you mark the free space one as experimental?

Reply:
@drcrallen my understanding from https://docs.oracle.com/javase/8/docs/api/java/io/File.html  is 
```
getTotalSpace() = total space the partition has
getUsableSpace() = space out of total that this user/vm can use , for example you can have cgroups restrictions in linux to limit space usage etc. haven't tested the theory though
getFreeSpace() = free space available
```

now all of the above are just hints and no guarantee about correctness in all cases and that is why I kept this feature completely optional so that users can explicitly use it for the file systems where it does what is intended.

how about updating the doc with the gotcha that" it depends on `getFreeSpace()` and make sure that works for your filesystem" rather than calling it experimental as I do think this is useful and not let the users shy away from using it.

Reply:
Ok, I'm good for that